docs/downloads/automation-library/integrations/sonar/label_sonar.cm

```diff
@@ -11,7 +11,7 @@ automations:
     run:
       - action: add-label@v1
         args:
-          label: '{{ item.icon }} x {{ item.count }} {{ item.name }}'
+          label: '{{ item.icon }} sonar:{{ item.name }}-{{ item.rating }}'
```

Comment:
Should we update the screenshots too in this case?

Reply:
yes... we should

Reply:
I'll create one

Reply:
Updated